pkg/server/node.go

```diff
@@ -812,6 +819,9 @@ func (n *Node) start(
 		log.Infof(ctx, "started with engine type %v", &t)
 	}
 	log.Infof(ctx, "started with attributes %v", attrs.Attrs)
+
+	n.startPeriodicLivenessCompaction(n.stopper,
+		livenessRangeCompactInterval.Get(&n.storeCfg.Settings.SV))
```

Comment:
I wonder if it might be better to arrange this such that the interval can be adjusted without restarting the node?